client/signup/steps/jpo-site-type/select-business-address.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { translate } from 'i18n-calypso';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+
+import PersonalGraphic from './personal-graphic';
+import BusinessGraphic from './business-graphic';
+
+module.exports = React.createClass( {
+
+	displayName: 'SelectBusinessAddress',
+
+	propTypes: {
+		required: React.PropTypes.bool,
+	},
+
+	render: function() {
+		if ( ! this.props.current ) {
+			return ( <div /> );
+		}
+
+		return ( 
+			<div className="jpo__site-type-wrapper business-address">
+				<Card>
+					<FormLabel>{ translate( 'Business Name' ) }</FormLabel>
+					<FormTextInput onChange={ this.props.onInputBusinessName } />
+					<FormLabel>{ translate( 'Street Address' ) }</FormLabel>
+					<FormTextInput onChange={ this.props.onInputStreetAddress } />
+					<FormLabel>{ translate( 'City' ) }</FormLabel>
+					<FormTextInput onChange={ this.props.onInputCity } />
+					<FormLabel>{ translate( 'State' ) }</FormLabel>
+					<FormTextInput onChange={ this.props.onInputState } />
+					<FormLabel>{ translate( 'ZIP Code' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Zip Code' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-flow-steps)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).